unit-tests/algo/thermal-loop/test-get-scale.cpp

```diff
@@ -0,0 +1,54 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+//#cmake:add-file ../../../src/algo/thermal-loop/*.cpp
+#include "../algo-common.h"
+#include "./create-syntetic-data.h"
+#include "algo/thermal-loop/l500-thermal-loop.h"
+
+using namespace librealsense::algo::thermal_loop::l500;
+
+
```

Comment:
I don't see any test for a table that has 0 for a scale

Reply:
Added test